static/app/views/replays/detail/focusArea.tsx

```diff
@@ -43,11 +44,18 @@ function FocusArea(props: Props) {
 
 function ActiveTab({active, replay}: Props & {active: ReplayTabs}) {
   const {routes, router} = useRouteContext();
+  const {setCurrentTime, setCurrentHoverTime} = useReplayContext();
   const organization = useOrganization();
 
   const event = replay.getEvent();
   const spansEntry = replay.getEntryType(EntryType.SPANS);
 
+  // Memoize this because re-renders will interfere with the mouse state of the
+  // chart (e.g. on mouse over and out)
+  const memorySpans = useMemo(() => {
+    return spansEntry?.data.filter(replay.isMemorySpan);
```

Comment:
We could also go for `replay.getRawSpans().filter(replay.isMemorySpan)`

Then the props inside `<MemoryChart>` would not have to deal with undefined: `memorySpans: MemorySpanType[];`